docs/content/en/docs/pipeline-stages/status-check.md

```diff
@@ -15,11 +15,10 @@ Commands that trigger a deployment, like `skaffold dev`, `skaffold deploy`, `ska
 Skaffold monitors the status of the following resource types:
 * [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/): check that the pod and its containers are in a `Ready` state.
 * [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/): check the output of `kubectl rollout status deployment` command 
-* [`Stateful Sets`](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/): check the output of `kubectl rollout status statefulset` command  
-* [`Google Cloud Config Connector resources`](https://cloud.google.com/config-connector/docs/overview): check that the resource state has a `Ready` condition set to `True`.
```

Comment:
Did you mean to remove Config Connector?

Reply:
yes, I thought that's what https://github.com/GoogleContainerTools/skaffold/pull/6898#discussion_r755345951 meant. Also it looked a bit odd listing that as a resource type alongside native types like pods, deployments and statefulsets.